src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/PrisonRegisterClient.kt

```diff
@@ -64,6 +66,19 @@ class PrisonRegisterClient(
   fun getPrison(prisonCode: String): PrisonRegisterPrisonDto = getPrisonAsMono(prisonCode)
     .blockOptional(apiTimeout).orElseThrow { NotFoundException("Prison with code - $prisonCode not found on prison-register") }
 
+  fun getPrisons(prisonCodes: List<String>): List<PrisonRegisterPrisonDto>? {
+    val uri = "/prisons/prisonsByIds"
```

Comment:
uses this endpoint to get multiple prisons at once - https://prison-register.hmpps.service.justice.gov.uk/swagger-ui/index.html#/prison-resource/getPrisonsByIds